projects/schematics/src/add-spartacus/index.ts

```diff
@@ -25,13 +25,16 @@ import {
   addImport,
   addToModuleImportsAndCommitChanges,
 } from '../shared/utils/module-file-utils';
-import { getAngularVersion } from '../shared/utils/package-utils';
+import {
+  getAngularVersion,
+  getSpartacusSchematicsVersion,
+} from '../shared/utils/package-utils';
 import { getProjectFromWorkspace } from '../shared/utils/workspace-utils';
 import { Schema as SpartacusOptions } from './schema';
 
 function addPackageJsonDependencies(): Rule {
   return (tree: Tree, context: SchematicContext) => {
-    const spartacusVersion = '^1.3';
+    const spartacusVersion = `^${getSpartacusSchematicsVersion()}`;
```

Comment:
As discussed, we should take into account pre-release idenfiers here, possbility just using full version here and just add ^ to it.